src/alphabet.rs

```diff
@@ -123,6 +123,11 @@ impl Alphabet {
 
         Ok(Self::from_str_unchecked(alphabet))
     }
+
+    /// Create a `&str` from the symbols in the `Alphabet`
+    pub fn as_str(&self) -> &str {
+        std::str::from_utf8(&self.symbols).unwrap()
```

Comment:
oh right... CI is mad because you can't use `std` in `no_std`. Just `str`, or the path using `core`, should work.